pytest_fixtures/component/provision_vmware.py

```diff
@@ -0,0 +1,62 @@
+from fauxfactory import gen_string
+import pytest
+
+from robottelo.config import settings
+from robottelo.constants import FOREMAN_PROVIDERS
+
+
+@pytest.fixture(scope='module')
+def module_vmware_cr(module_provisioning_sat, module_sca_manifest_org, module_location):
+    vmware_cr = module_provisioning_sat.sat.cli.ComputeResource.create(
```

Comment:
Can we use API call here as its setup fixture?

Reply:
Updated